chart/templates/worker-budget.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ .Release.Name }}-worker-budget
+spec:
+  minAvailable: 1
+  selector:
+    matchLabels:
+      app: {{ .Release.Name}}
+      tier: worker
```

Comment:
Same problem as the web PDB: it has app: svc-questionnaires hardcoded, maybe change that there

Reply:
Fixed.